playbook/app/pb_kits/playbook/pb_bread_crumbs/_bread_crumbs.jsx

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+
+import React from 'react'
+import PropTypes from 'prop-types';
+import classnames from 'classnames'
+import { globalProps } from '../utilities/globalProps.js'
+
+import {
+  buildAriaProps,
+  buildCss,
+  buildDataProps,
+} from '../utilities/props'
+
+type BreadCrumbsProps = {
+  aria?: object,
+  className?: string,
+  data?: object,
+  id?: string,
+  text?: string
+}
+const BreadCrumbs = (props: BreadCrumbsProps) => {
+  const {
+    aria = {},
+    className,
+    data = {},
+    id,
+    children
+  } = props
+  const ariaProps = buildAriaProps(aria)
+  const dataProps = buildDataProps(data)
+  const css = classnames(
+    buildCss('pb_bread_crumbs_kit'),
+    globalProps(props),
+    className
+  )
+
+  return (
+    <div
```

Comment:
```suggestion
    <nav
```

Reply:
Adding a [HTML Landmark](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html) here makes sense for accessibility